Lombiq.NodeJs.Extensions/pnpm-lock.yaml

```diff
@@ -1,5384 +0,0 @@
-lockfileVersion: 5.3
```

Comment:
Why is this not needed anymore?

Reply:
I was mistaken, apparently NodeJs.Extensions installs the packages in a different location (in the solution root). So the lock file should be there too.

Reply:
I'm confused - why does this file appear to be deleted when currently the [`dev` branch](https://github.com/Lombiq/NodeJs-Extensions/tree/dev/Lombiq.NodeJs.Extensions) does not contain it?